force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+        String greeting = 'Hello, Salesforce!';
         
         // Get the length of the String        
+        Integer lengthOfGreeting = greeting.length();
+
         // Get a substring of the String of the first 5 characters
+        String first5 = greeting.substring(0, 5);
+
         // Convert the String to lower case
+        String lowerCaseGreeting = greeting.toLowerCase();
         
         // Return the list with the string data
+        String [] greetingList = new List <String>();
+
         // In the first position add the string length
+        greetingList.add(lengthOfGreeting.toString());        
         // In the second position add the substring
+        greetingList.add(first5);
         // In the third position add the string in lower case
+        greetingList.add(lowerCaseGreeting);
```

Comment:
Try to use the addAll() function to all all these three to the list at the same time.